db/re/item_db_equip.yml

```
@@ -110468,6 +111644,244 @@ Body:
             bonus2 bSkillCooldown,"RL_H_MINE",-1000;
          }
       }
+  - Id: 28227
+    AegisName: Diva_Rifle
+    Name: Diva Rifle
+    Type: Weapon
+    SubType: Rifle
+    Weight: 850
+    Attack: 170
+    Range: 9
+    Slots: 1
+    Jobs:
+      Rebellion: true
+    Locations:
+      Both_Hand: true
+    WeaponLevel: 4
+    EquipLevelMin: 150
+    Refineable: true
+    Script: |
+      .@r = getrefine();
+      bonus2 bSkillAtk,"RL_HAMMER_OF_GOD",(7*readparam(bInt)/10);
```

Precedence, the result is different
```suggestion
      bonus2 bSkillAtk,"RL_HAMMER_OF_GOD",(readparam(bInt)/10)*7;
```

---

Thanks